server/views/components/iframe/iframe.njk

```diff
@@ -1,7 +1,19 @@
-<div class="iframe-container">
-  <iframe class="iframe-container__iframe"
-    src="{{ model.src }}"
+<div class="iframe-container relative js-iframe-container">
+  <button class="iframe-container__trigger plain-button no-padding no-visible-focus absolute js-iframe-trigger"
+    data-track-event='{"category": "component", "action": "launch-iframe:click", "label": "iframeSrc:{{ model.src }}"}'>
+    <div class="iframe-container__overlay absolute"></div>
+    <span class="iframe-container__launch absolute btn js-iframe-launch">
+      {{ model.launchButtonText }}
```

Comment:
Would this be better as a play / launch button, something consistent for users to get used to?

Reply:
Just think you may want to signal what's going to happen when clicked

Reply:
See above.